prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/CustomFHIRFunctionsTests.kt

```diff
@@ -258,8 +299,29 @@ class CustomFHIRFunctionsTests {
             )
         ).isEmpty()
 
-        val parts = CustomFHIRFunctions.split(mutableListOf(stringToSplit), mutableListOf(mutableListOf(delimiter)))
+        val parts = CustomFHIRFunctions.split(
+            mutableListOf(stringToSplit),
+            mutableListOf(mutableListOf(delimiter))
+        )
         assertThat(parts).isNotEmpty()
         assertThat(parts.size).isEqualTo(3)
     }
+
+    @Test
+    fun `test livdTableLookup is Observation`() {
+        assertThat(
+            CustomFHIRFunctions.livdTableLookup(
+                mutableListOf(Observation()), mutableListOf(), UnitTestUtils.simpleMetadata
+            ) == mutableListOf(StringType(null))
+        )
+    }
+
+    @Test
+    fun `test livdTableLookup is not Observation`() {
+        assertThat {
+            CustomFHIRFunctions.livdTableLookup(
+                mutableListOf(Device()), mutableListOf(), UnitTestUtils.simpleMetadata
+            )
+        }.isFailure().hasMessage("Must call the livdTableLookup function on an observation")
```

Comment:
This is a brittle test. Remove the has message portion as any small change will break this. You just need to know that an exception is thrown.